src/ngx_postgres_rewrite.c

```diff
@@ -187,7 +187,7 @@
         url_variable.len = 0;
         //fprintf(stdout, "something here %s\n", p);
 
-        while(url_variable.len < (redirect + size) - (p + 1)) {
+        while((int)url_variable.len < (redirect + size) - (p + 1)) {
```

Comment:
@fridjon no no no. You can just continue commiting to you repo. Github will notice these changes.